pycm/pycm_curve.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""Curve module."""
+from __future__ import division
+from .pycm_error import pycmCurveError
+from .pycm_util import threshold_func, thresholds_calc, isfloat
+from .pycm_param import *
+from .pycm_obj import ConfusionMatrix
+import numpy
+
+
+class Curve:
+    """
+    Curve class.
+    """
+
+    def __init__(self, actual_vector, prob_vector, classes, thresholds=None, sample_weight=None):
+        """
+        Init method.
+
+        :param actual_vector: actual vector
+        :type actual_vector: python list or numpy array of any stringable objects
+        :param prob_vector: vector of probabilities
+        :type prob_vector: python list or numpy array of any stringable objects
```

Comment:
I suggest `prob` instead of `prob_vector`. The second one somehow reminds me of what we have in the `ConfusionMatrix.__init__` as `pred_vector` but it is not a true intuition.

Reply:
Good idea!